composer.json

```diff
@@ -10,13 +10,13 @@
         }
     ],
     "require": {
-        "zendframework/zend-http": "~2.2.5",
-        "zendframework/zend-stdlib": "~2.2.5",
-        "zendframework/zend-uri": "~2.2.5"
+        "php": ">=5.5.0",
+        "zendframework/zend-stdlib": "~2.2.5"
     },
     "require-dev": {
         "phpunit/PHPUnit": "3.7.*",
         "satooshi/php-coveralls": ">=0.6.0",
+        "psr/http-message": "1.0.*@dev",
```

Comment:
It should be explicit. Consider that `psr/http-message` would have multiple versions. How can I know which version of psr-7 you want me to implement?